### PR TITLE
add app css

### PR DESCRIPTION
**Pull request by mikbry**
#44

### CODE REVIEW

**Comment by mikbry** [comment]
I made some tests for theming also.
And fix crash at dev start with wrong message

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "clean-dist": "rm -r -f dist && shx mkdir -p dist/public/css",
     "compile": "yarn compile:css && babel src/shared -d dist && cp -f ./package.json ./dist/package.json && cp -f ./LICENSE ./dist/LICENSE && cp -f ./README.md ./dist/README.md",
-    "compile:css": "yarn clean-dist && cleancss -o dist/public/css/compressed.css node_modules/zrmc/compressed.css node_modules/zoapp-ui/compressed.css",
+    "compile:css": "yarn clean-dist && cleancss -o dist/public/css/compressed.css src/public/css/app.css node_modules/zrmc/compressed.css node_modules/zoapp-ui/compressed.css",
```
Comment (willdurand) [inline package.json]: shouldn't we put the app style at the end?

File: src/client/index.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ const app = {
 const front = new Front("app", app, config);
 
 front.start(true);
-front.store.dispatch(setMessage({ message: "Welcome Dude !" }));
+front.store.dispatch(setMessage("Welcome Dude !"));
```
Comment (willdurand) [inline src/client/index.js]: this has changed recently I guess. can we update the content to "Welcome Human!"?

**Comment by mikbry** [comment]
In fact it is not ok. The compressed.css is located in dist/public/css/ not dist/ as in zrmc.

**Comment by willdurand** [comment]
doh

**Comment by willdurand** [comment]
well actually I dont understand, you mean the zoapp/front compressed CSS file?

**Pull request merged**